### ISSUE TITLE
Does it work?

### ISSUE THREAD

**Issue by lahwaacz** [quarto-dev/jupyterlab-quarto#6]
I created a clean virtual environment where I installed JupyterLab and Quarto:
```
pip install jupyterlab jupyterlab-quarto
```

However, it still does not seem to work - notably the frontmatter is not rendered as in the [screenshot in README](https://github.com/quarto-dev/jupyterlab-quarto?tab=readme-ov-file#jupyterlab-quarto-extension):

![screenshot-2024-02-17@10:01:33](https://github.com/quarto-dev/jupyterlab-quarto/assets/1289205/633c5a8b-474f-44d3-979a-3ae396aaf997)

**Comment by dragonstyle** [comment]
Hello - thanks for opening this issue! It looks to me like the front matter cell is a `raw` cell (which isn't rendered as markdown in jupyterlab). Try switching that cell to a markdown cell and things should work as you'd expect.

Let us know if that clears this up!

**Comment by dragonstyle** [comment]
You're correct- we'll either update that doc or I'll figure out how to make raw cell rendering working in a JL extension...

**Comment by tmatthias29** [comment]
Might be good to change the documentation here, or at least leave this issue open, until you fix this or chnange the Quarto docs page. I had the same issue as the OP and had to open a closed issue to find the solution.

**Comment by dragonstyle** [comment]
Good suggestion, I re-opened this. A couple of notes from my research:

- We currently document using a raw block as the front matter in our documentation largely because for viewers of a notebook without the extension installed, this will look significantly better. It is unrendered, but at least you can see it is front matter. With a non-raw block, It just ends up a jumble of badly rendered markdown (since the renderer doesn't recognize the raw block).
- I looked into a rendering the raw block, but I could only find a way to _completely_ take over rendering of the raw block (including managing edit vs non-edit mode, etc...). I wasn't quite willing to take a bit that big, so we ended up here for now. This branch https://github.com/quarto-dev/jupyterlab-quarto/tree/feature/raw-cell contains the proof of concept.
